src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -490,20 +501,29 @@ def crop_augmenter(
     return np.array(image), annotation
 
 
-def complete_augmenter(img_tf, ann_tf, output_height, output_width):
-    img_tf, ann_tf = tf.numpy_function(
-        func=crop_augmenter, inp=[img_tf, ann_tf], Tout=[tf.float32, tf.float32]
-    )
-    img_tf, ann_tf = tf.numpy_function(
-        func=padding_augmenter, inp=[img_tf, ann_tf], Tout=[tf.float32, tf.float32]
-    )
+def numpy_augmenter(img, ann, seed):
+    random = np.random.RandomState(seed=seed)
+    img, ann = crop_augmenter(img, ann, random)
+    img, ann = padding_augmenter(img, ann, random)
+    img, ann = horizontal_flip_augmenter(img, ann, random)
+    return img, ann
+
+
+def complete_augmenter(
+    img_tf,
+    ann_tf,
+    seed_tf,
+    alpha_tf,
+    output_height,
+    output_width
+):
     img_tf, ann_tf = tf.numpy_function(
-        func=horizontal_flip_augmenter,
-        inp=[img_tf, ann_tf],
-        Tout=[tf.float32, tf.float32],
+        func=numpy_augmenter,
+        inp=[img_tf, ann_tf, seed_tf],
+        Tout=[tf.float32, tf.float32]
```

Comment:
Major nit: I actually generally prefer keeping trailing comma. From [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas):
```
When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line. 
```

Reply:
I honestly did not consciously intend to remove that comma! I actually to punctuate in this fashion too (for languages/syntaxes that support it), for the same reason